### ISSUE TITLE
Import des fonts mal placé

### ISSUE THREAD

**Issue by kewinMarchand** [kewinMarchand/wp-rest#49]
L'import de la font se fait dans PageLayout qui est un fichier JS, il sera chargé après React, ça peut prendre un moment.

La balise Helmet permet d'ajouter des metas dynamiques, par exemple le title ou la description qui sont administrables dans le BO Wordpress.

La font n'étant pas dynamique, il faut la charger dans le fichier index.html, elle sera chargée bien plus vite.

**Comment by kewinMarchand** [comment]
Y'a pas de PR associée, cette issue peut pas être completed

**Comment by ThomasTSWD** [comment]
Je vois pas comment faire la. Je dois recommencer non ? je crée une nouvelle branche ?

**Comment by kewinMarchand** [comment]
@ThomasTSWD oui, il faut créer une branche, supprimer l'mport du js et le mettre dans le html

**Comment by ThomasTSWD** [comment]
Je créée une branche du main n'est-ce pas ? Pas de ma branche en erreur ?

**Comment by kewinMarchand** [comment]
yes et oublie pas de pull avant, on a mergé des PR

**Comment by ThomasTSWD** [comment]
normalement c bon. J'ai mis la font poppins pour tester. Sur yarn ok

**Comment by ThomasTSWD** [comment]
j'ai commit et push sur la branche

**Comment by ThomasTSWD** [comment]
Excue moi j'avais créé une branche fonts new avec une nouvelle typo pour tester. Tu as recu une demande pour le mettre sur master ?

**Comment by kewinMarchand** [comment]
y'a une branche ici: https://github.com/kewinMarchand/wp-rest/tree/fonts_new mais pas de PR ouverte. Attention, il faut la rebase.

**Comment by kewinMarchand** [comment]
Par contre t'as changé la font mais elle est tjrs pas importée au bon endroit :)

**Comment by ThomasTSWD** [comment]
ah pourtant je l'avais remis dans le html .. mais au 2e commit ^^'
Les checks avaient passé. 
Je regarde, merci !

**Comment by ThomasTSWD** [comment]
j'ai recommencé depuis le master

**Comment by kewinMarchand** [comment]
@ThomasTSWD il faut le lien de ta PR dans l'issue pour pouvoir tracer tout ça :)
